spec/models/manageiq/providers/lenovo/physical_infra_manager/physical_infra_spec_common.rb

```diff
@@ -0,0 +1,35 @@
+module PhysicalInfraSpecCommon
+  extend ActiveSupport::Concern
+
+  MODELS = %i(
+    ext_management_system physical_rack physical_chassis physical_server
+    physical_switch physical_storage
```

Comment:
does this list all models? I think e.g. physical_disk is missing

Reply:
No, I forgot to put it after physical_storage, sorry about that. Now it's fixed.